### PR TITLE
crew: Update `git` command used in `crew sysinfo`

### PR DESCRIPTION
**Pull request by supechicken**
Seems the `git` command that uses to get the latest commit time does not work on every system, weird...

### CODE REVIEW

**Comment by satmandu** [comment]
Where does it break?

**Comment by satmandu** [comment]
Seems that we REALLY want the last update time though, no?

Could this just be due to really out of date chromebrew setups?

**Comment by supechicken** [comment]
> Seems that we REALLY want the last update time though, no?
> 
> Could this just be due to really out of date chromebrew setups?

But #7224 is from 24 days ago...

Finding another way to get the latest commit time...

**Comment by satmandu** [comment]
Something like `git show -s --format=%H` would show the last commit...

**Comment by supechicken** [comment]
> Something like git show -s --format=%H would show the last commit...

It also works with colors :)
```shell
git show --color=always -s --format='%C(yellow)%h%C(reset)  %s  %C(brightcyan)(%cr)%C(reset)' | cat
```

**Comment by supechicken** [comment]
Added a few changes and now it looks like this:
- Architecture: `x86_64` (`x86_64`)
- Kernel version: `5.10.120-16471-g167ad2e5dcba`

- Chromebrew version: `1.24.1`
- Chromebrew prefix: `/usr/local`
- Chromebrew libdir: `/usr/local/lib64`

- Last update in local repository: 8ae1b0a `Remove color as markdown does not support (70 seconds ago)`

- OS variant: `Chrome OS`
- OS version: `reven-release/R104-14909.100.0`
- OS channel: `stable-channel`

File: bin/crew [hunk 1]
```diff
@@ -1863,7 +1863,7 @@ def sysinfo_command(args)
     - Chromebrew prefix: `#{CREW_PREFIX}`
     - Chromebrew libdir: `#{CREW_LIB_PREFIX}`
 
-    - Last update in local repository: `#{`git show -s --format='#{git_commit_message_format}' '#{CREW_LIB_PATH}'`.chomp}`
+    - Last update in local repository: #{`git show -s --format='#{git_commit_message_format}' '#{CREW_LIB_PATH}'`.chomp}
```
Comment (uberhacker) [inline bin/crew:1866]: Is the outer `#{...}` necessary?

File: bin/crew [hunk 2]
```diff
@@ -1853,7 +1853,7 @@ def sysinfo_command(args)
     lsb_release = ENV
   end
 
-  git_commit_message_format = '%h  %s  (%cr)'
+  git_commit_message_format = '%h `%s (%cr)`'
```
Comment (uberhacker) [inline bin/crew:1856]: Why the backticks?

**Review by uberhacker — CHANGES_REQUESTED** [review]

File: bin/crew (hunk 2 above)
Comment (supechicken) [reply bin/crew:1856]: It is for markdown styling (\`abc\` shows `abc`)

File: bin/crew (hunk 1 above)
Comment (supechicken) [reply bin/crew:1866]: Seems this syntax highlight of github is a bit misleading😅, maybe this more clear:
```ruby
"- Last update in local repository: #{IO.popen("git show -s --format='#{git_commit_message_format}' '#{CREW_LIB_PATH}").read.chomp}"

File: bin/crew (hunk 2 above)
Comment (uberhacker) [reply bin/crew:1856]: Ummm, I don't think that is going to work like you might think.   Backticks are used to EXECUTE commands.

Comment (supechicken) [reply bin/crew:1856]: This is not `bash`... In ruby only `#{}` in `""` will be executed (that means the \` does not have any special meaning in strings)

Comment (supechicken) [reply bin/crew:1856]: Oh I see what you mean😅 The `#{git_commit_message_format}` in the git command is wrapped with `''` so it will not be executed

Comment (uberhacker) [reply bin/crew:1856]: Backticks work in ruby too.  I think you are right about the single quotes.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**